tests/pylint/runpylint.py

```diff
@@ -16,7 +21,8 @@ def __init__(self):
                                 # TODO: Should be fixed with https://github.com/PyCQA/astroid/pull/433
                                 FalsePositive(r"^E1129.*: Context manager 'lock' doesn't implement __enter__ and __exit__.$"),
 
-                                # XXX: This is temporary until koji has python3 versions.
+                                # We have koji used only in make_updates script there is no need to
+                                # add more test dependencies because of this script
```

Comment:
I don't understand the comment.

Reply:
It is not a temporary fake positive. We don't use this dependency in Anaconda anywhere and I don't think we want to have a new build requires in spec file only because it is used in `make_updates`. However the problem is that pylint is run on all the scripts in so also on `make_updates`.

Reply:
Couldn't we just disable pylint for `import koji` with a comment? Do we need to have a false positive here?

Reply:
True enough, ok I'll change that.